lib/parse.js

```diff
@@ -10,6 +10,12 @@ var DOMParser = require('xmldom').DOMParser;
 
 exports.parse = parse;
 
+var NODE_TYPES = {
```

Comment:
Otherwise we could just disable them for PRs - I think they'll still be run on master merge, see this section: https://docs.travis-ci.com/user/pull-requests#Pull-Requests-and-Security-Restrictions